Client/Frontend/Widgets/ActionOverlayTableViewController.swift

```diff
@@ -155,10 +160,18 @@ class ActionOverlayTableViewHeader: UITableViewHeaderFooterView {
         let siteImageView = UIImageView()
         siteImageView.contentMode = UIViewContentMode.ScaleAspectFit
         siteImageView.clipsToBounds = true
-        siteImageView.layer.cornerRadius = SimpleHighlightCellUX.CornerRadius
         return siteImageView
     }()
 
+    lazy var siteImageBackground: UIView = {
```

Comment:
I thought that setting a `UIImageView` frame size to `32 px` was optimal performance-wise than manually rescaling the `UIImage` to `32 px`. Hence, I embedded the `UIImageView` (of size `32 px`) in a `UIView (siteImageBackground)`. `drawInRect` (as used in `SimpleHighlightCell`) seems a costly affair.